notebooks/data_viz_easy/track_meta.py

```diff
@@ -0,0 +1,28 @@
+# See also examples/example_track/track_meta.py for a longer, commented example
+track = dict(
+    author_username='',
```

Comment:
It's worth putting your kaggle username here.

It will get filled in for track_metadata.py files... which allow you to push the kernels to kaggle when the time comes